docs-source/content/userguide/cicd/choose-an-approach.md

```diff
@@ -10,44 +10,69 @@ Let's review what we have discussed and talk about when we might want to use
 various approaches.  We can start by asking ourselves questions like these:
 
 
-- *Can you make the desired change with a configuration override?*  
-  The WebLogic Kubernetes Operator allows you to inject a number of [configuration
+- *Can you make the desired change with a configuration override or Model in Image config map?*  
+
+  When your domain source type is Domain in PV or Domain in Image,
+  the WebLogic Kubernetes Operator allows you to inject a number of [configuration
   overrides]({{< relref "/userguide/managing-domains/configoverrides/_index.md" >}})
-  into your pods before starting any servers in the domain.  This allows you to use 
-  the same image for multiple
-  different configurations.  A good example would be changing the settings for a data
-  source, for example. You may wish to have a larger connection pool in your production
+  into your pods before starting any servers in the domain.  
+
+  When your domain
+  source type is Model in Image you can 
+  [inject model updates]({{< relref "/userguide/managing-domains/model-in-image/runtime-updates.md" >}})
+  before starting any servers in the domain for any type of update, or, even while your domain
+  is running for most types of updates. Model in Image runtime model updates are propagated
+  by [restarting]({{< relref "/userguide/managing-domains/domain-lifecycle/restarting.md" >}})
+  (rolling) the running WebLogic servers.
+
+  A good example of a change would be changing the settings for a data
+  source. For example, you may wish to have a larger connection pool in your production
   environment than you do in your development/test environments.  You probably also
   want to have different credentials.  You may want to change the service name, and
-  so on.  All of these kinds of updates can be made with configuration overrides.
+  so on. All of these kinds of updates can be made with configuration overrides
+  for Domain in PV and Domain in Image, and with model updates for Model in Image.
   These are placed in a Kubernetes config map, that is, they are outside of the image, so
   they do not require rebuilding the Docker image.  If all of your changes fit into
   this category, it is probably much better to just use configuration overrides
```

Comment:
Not sure if this sentence still makes sense here, "If all of your changes fit into this category, it is probably much better to just use configuration overrides instead of building a new image."

Reply:
Oops.  Good catch.  Cut and paste error.  I'll fix that.

Reply:
@rosemarymarano I pushed a fix. Please resolve if you think its fine now, otherwise please comment.

Reply:
@tbarnes-us Not seeing the fix.

Reply:
@rosemarymarano Hmm.  I messed up in git somehow.  I started with a fresh branch and pushed a new change.  You should see it now.

Reply:
I do.